client/landing/gutenboarding/components/domain-picker/button.tsx

```diff
@@ -65,28 +67,27 @@ const DomainPickerButton: FunctionComponent = () => {
 	const [ freeDomainSuggestions, paidDomainSuggestions ] = partition( suggestions, 'is_free' );
 
 	return (
-		<Button
-			className="domain-picker__button"
-			onClick={ () => setDomainPopoverVisibility( s => ! s ) }
-		>
-			<div className="domain-picker__site-title">
-				{ siteTitle ? siteTitle : NO__( 'Create your site' ) }
-			</div>
-			<div>{ head( freeDomainSuggestions )?.domain_name }</div>
+		<>
+			<Button
+				className={ classnames( 'domain-picker__button', { 'is-open': isDomainPopoverVisible } ) }
+				aria-pressed={ isDomainPopoverVisible }
```

Comment:
https://www.w3.org/TR/wai-aria-practices/#button

`aria-hasmenu` and `aria-expanded`.